extension.html

```diff
@@ -141,17 +136,21 @@ <h2>Extensions to the <code>PointerEvent</code> interface</h2>
                   <p>Returns a sequence of <code>PointerEvents</code> the user agent predicts that will follow the sequence of coalesced events in <code>pointermove</code> event in the future.
 The number of events returned by this API and how far they are from the current timestamp are determined by the user agent and the prediction algorithm it uses.
 This API always returns an empty list for pointerevent types other than <code>pointermove</code>.
-Similar to the coalesced events the following attributes of the predicted events will always have the same value as the dispatched event:
-<code>pointerId</code>, <code>pointerType</code>, <code>isPrimary</code>, <code>isTrusted</code>, <code>target</code>.
-Also <code>cancelable</code> and <code>bubbles</code> attributes will be false and other attibutes related to the <a href="https://www.w3.org/TR/uievents/#event-flow">event dispatch algorithm</a>
-(e.g. <code>currentTarget</code>, <code>eventPhase</code>) will have their default value
-and the related functions (e.g. <code>stopPropagation</code>, <code>stopImmediatePropagation</code>) will do nothing.
-Note that since the <code>target</code> of the predicted events is the same as the dispatched event's <code>target</code>,
-the attributes like <code>offsetX</code>, <code>offsetY</code> ([[!CSSOM-VIEW]]) which are commonly used by drawing applications will be calculated correctly for the predicted events.
+Similar to the coalesced events, the following attributes of the predicted events will always have the same value as the dispatched event:
+<code>pointerId</code>, <code>pointerType</code>, <code>isPrimary</code>, <code>isTrusted</code>.
+<code>cancelable</code> and <code>bubbles</code> attributes will be false.
 The rest of the attributes might be different from the dispatched event based on the prediction algorithm in user agent.
 The events in the sequence will have increasing <a href="https://dom.spec.whatwg.org/#dom-event-timestamp"><code>timeStamps</code></a> ([[!WHATWG-DOM]]).
 So the first event will have the smallest <code>timeStamp</code> which should still be greater than the last event in the coalesced events.</p>
               </dd>
+
+              <p>
+                The <code> target </code> attribute's getter of both coalesced events and predicted events, when invoked, must return the associate dispatched event's target.
```

Comment:
I'm not sure if this addresses @annevk concerns. Although you did change the text to talk about the target's getter and hence that might be good enough.

But let's ask @annevk. Does this look okay or do you also want a note in https://dom.spec.whatwg.org/#dom-event-target mentioning this spec might override the getter?

Reply:
I think it needs to be more than a note, it needs to be part of the model, right? You want to define some kind of extensibility hook that you use here.

Reply:
When you say an "extensibility hook", do you not consider the Event.target as a hook that can be extended? Like what this [PR](https://github.com/EiraGe/dom/pull/1) refers to. Do you want like an explicit function like getTarget that is returning the value of target attribute and that can be overridden here instead or something like that?

Reply:
Yeah, `Event.prototype.target` is a public API you cannot really touch. You need to add some kind of internal-facing operation you can override in other specifications, which also makes it clear from the DOM Standard that this is a thing some specifications do (it should probably have a note pointing them out, only this one so far).

Reply:
I updated this and also the other [PR](https://github.com/EiraGe/dom/pull/1) for DOM. Is this clearer now?